<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_simple">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 211, 189);</string>
           </property>
           <property name="text">
            <string>Simple Pass</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_Extra">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 150, 97);</string>
           </property>
           <property name="text">
            <string>Extra Pass</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_superExtra">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 90, 78);</string>
           </property>
           <property name="text">
            <string>Super Extra Pass</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
